### ISSUE TITLE
Refreshing gamelist freezes PS3 when "Refresh XML: OK xxx Games" is displayed

### ISSUE THREAD

**Issue by Adamanimus** [aldostools/webMAN-MOD#778]
I don't know how long it had this problem but 1.47.42 freezes my PS3 everything I refresh gamelist. Can you look into it?

**Comment by aldostools** [comment]
It could be caused by a corrupted game, a bad file name, too much games or an error in the file system. It also may be a memory issue.

Disable VshFpsCounter if have it running before scan the games.

Unplug all USB devices to determine if some gane is causing the freeze. Also rename PS3ISO, PSXISO, ROMS in the internal HDD. The refresh XML. If it doesn't freeze rename back the folders and plug the USB devices progressively one by one until you identify the game causing the freeze. 

Rebuild the database in case it has some corruption. 

You can try an older version of wMM in case some recent change could be causing the bug. Or try uninstalling and reinstalling the latest build of  wMM 1.47.42

Never run a game while the system is still scanning the games.

**Comment by Adamanimus** [comment]
Hi, Aldo. I have tried everything you mentioned. I downgraded to 1.47.41 and it doesn't freeze but 1.47.42 freezes even when freshly installed.

**Comment by aldostools** [comment]
If you are using FPS counter, I recommend to avoid it if you're having freeze issues.

Delete /dev_hdd0/tmp/wm_config.bin and reboot to use the default settings.

Disconnect all USB devices and rescan to determine if the issue is related to some external content.

If the problem persists, keep using the previous version. There aren't major changes in features between these versions.

**Comment by winebottle61** [comment]
Hello, I'm writing here about my experience with this issue and how it ended up working out for me in case the information helps anyone. Sorry in advance if I missed some important notes. 

I'm running on the latest Hybrid firmware with the latest version of HEN on a PS3 Super Slim with the latest version of webMAN Mod

TLDR: Below

So in the beginning everything worked perfectly fine. I only had multiMAN and webMAN installed alongside some other homebrew (Apollo save editor, HDD space unlocker, etc) and the only "weird" Thing was that sometimes (I think after I started using ISO games) this new option would appear after I activate HEN alongside webMAN. It was a folder  with then name 
 "(staricon)/app_home/PS3_GAME/" but it only gave an error when I clicked on it. When it did appear, I just clicked on remote play and out again to refresh the XMB and then it would be gone. 

I thought I had the Lite version of webMAN and wanted to get the full version so I used the built in Uninstaller and then reinstalled it again using the same installation as the first time around (Turned out I had the full version all along). 

This is when the issues began. 

After the reinstall I would try to use the Refresh webMAN Games & XMB option and it would always freeze after it was complete (Right before it would normally click me back into the webMAN mod folder). I also noticed that the the star folder     ("(staricon)/app_home/PS3_GAME/") Would never appear again in the XMB. 

I have no game ISO of any kind on my internal HDD, I had them only on my external HDD however the freezes would occur regardless of whether my external HDD was connected or not.

I tried uninstalling and reinstalling multiple times. Deleted the original Installation and downloaded a new one from Brewology however the problem would persist none the less. I even followed the official uninstalling webMAN mod wiki to the best of my abilities to no avail. 

I uninstalled all homebrew applications (Not PKG games) and reinstalled OFW, messed around in the PS Store and synced my trophies and then went ahead and Jailbroke My PS3 Again using the same methods as before. Installed only multiMAN and webMAN. Realized while in multiMAN that folders related to other Homebrew were still here and therefore the webMAN issue is probably still here as well. I was correct. 

I messed around a bit, Noticed that the option to Refresh webMAN games would NOT freeze the system and the option to reload XMB would just give an error instead of refreshing XMB. Freezing only happened when choosing to refresh Games & XMB. I tried rebuilding the database but that didn't fix the issue.

At this point I googled and read your advice and did the exact following:
*Uninstalled webMAN. 
*renamed EVERY ISO folder including DVD and BD via multiMAN.
*reinstalled webMAN again from the same installation. 
*Noticed that the "(staricon)/app_home/PS3_GAME/" folder appeared again after enabling HEN like how it used to do sometimes before.
*Disconnected everything form all USB ports (Controller charging cable and External HDD). 
*Clicked to refresh on webMAN games & XMB
*SUCCESS!!!!!?????? 

I have no idea why this worked since all my ISO folders were empty and my ISO games were on the external HDD. webMAN created new ISO folders and detected my games from the external HDD just fine without freezing when I did connect my external HDD. Furthermore the option to reload XMB works now as well.  

So yea... Just try to rename the folders... IDK why.

TLDR: The issues that started after uninstalling and reinstalling webMAN were resolved after renaming all the ISO folders in dev/HDD0 even though they contained no games / movies what so ever. 

Sorry for the long Post, I just wanted to be as thorough as possible. 

P.S: Thank you and everyone who contributed to this application. Made my life way easier.
